lib/types/types.ts

```diff
@@ -15,6 +15,7 @@ export type committeePreferenceType = {
 };
 
 export type applicantType = {
+  _id: ObjectId;
```

Comment:
Hvorfor?

Reply:
Må ha en unique identifier når vi viser søkerne som er anonym.

Reply:
Åja, for å ikke bruke owId?

Reply:
Jepp, kan i teorien spores tilbake til brukeren